src/Compilers/Core/Portable/Syntax/SyntaxToken.cs

```diff
@@ -78,7 +78,7 @@ private string GetDebuggerDisplay()
 
         internal int Index { get; }
 
-        internal int Position { get; }
+        public int Position { get; }
```

Comment:
💭 It may be easier to get this accepted as:

```csharp
public int FullSpanStart => Position;
```

Reply:
(Agreed).